Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -612,8 +612,8 @@ rateCardConfig state =
         , alertDialogPrimaryColor = state.data.config.alertDialogPrimaryColor
         , description = if state.data.rateCard.nightCharges then (getString NIGHT_TIME_CHARGES) else (getString DAY_TIME_CHARGES)
         , buttonText = Just if state.data.rateCard.currentRateCardType == DefaultRateCard then (getString GOT_IT) else (getString GO_BACK_)
-        , driverAdditionsImage = if (MU.getMerchant FunctionCall == MU.YATRI) then "ny_ic_driver_additions_yatri,https://assets.juspay.in/beckn/yatri/user/images/ny_ic_driver_additions_yatri.png" 
-                                                                              else "ny_ic_driver_addition_table2,https://assets.juspay.in/beckn/nammayatri/user/images/ny_ic_driver_addition_table2.png"
+        , driverAdditionsImage = if (MU.getMerchant FunctionCall == MU.YATRI && state.data.rateCard.vehicleVariant /= "AUTO_RICKSHAW") then "ny_ic_driver_additions_yatri,https://assets.juspay.in/beckn/yatri/user/images/ny_ic_driver_additions_yatri.png" 
```

Comment:
@Anuragini-2506 can we use this from configs, instead of using a merchant check.

Reply:
Will do this changes and raise a separate PR for refactoring the vehicleType in driver and customer side